source/_patterns/50-pages/40-demopages/rlp-elections-header.twig

```diff
@@ -0,0 +1,36 @@
+{% embed 'templates-landingpage' %}
+    {% block landingpage_content %}
+        <div class="collection collection--standard collection--with-title-button">
+            <div class="collection__branding-wrapper js-collection-branding-wrapper">
+                <div class="collection__content-wrapper js-collection-content-wrapper">
+                    <h3 class="collection__title">
+                        <span class="h-loading-only">
+                            <span class="shimmer shimmer--heading shimmer--shorter"></span>
+                            <span class="shimmer shimmer--heading shimmer--short"></span>
+                            <span class="shimmer shimmer--heading shimmer--short"></span>
+                        </span>
+                        <span class="collection__title-logo"><img src="{{ frameworkAssetPath }}/img/flags/cantons/ag.svg" alt="Wappen AG"></span>
+                        <span class="collection__title-text">Aargau</span>
+                        <a href="#" class="collection__title-button">
+                            <span class="collection__title-button-text">Kanton ändern</span>
+                            <span class="collection__title-button-text-mobile" aria-hidden="true">Ändern</span>
```

Comment:
Was spricht gegen eine Zeile mit dem «Kanton» in einem Container, der nur auf Tablet+ angezeigt wird?

Reply:
ich hatte bedenken wegen der screenreader. im prinzip müsste ich dann "kanton" mit den css-rules von "h-offscreen" verstecken, damit screenreader user noch "drauskommen", was etwas mehr css ist. fand das etwas transparenter für uns entwickler.

Reply:
Noch als nachträglicher Input 😄:
Zwei verschiedene Elemente machen hier schon Sinn, denn
* so muss man nicht mental Strings zusammensetzen
* man muss nicht mit Leerzeichen oder Gross-/Kleinschreibung aufpassen
* hat man keine Probleme wenn sich der Text ändert und nicht mehr einfach halbieren lässt
* hat man keine Probleme bei Translations (wo z.b. die Reihenfolge anders ist)